Sources/SwiftProtobuf/Message.swift

```diff
@@ -96,15 +117,22 @@ public protocol Message: CustomDebugStringConvertible {
   // JSON encoding/decoding support
   //
 
-  /// Returns a JSON-coded representation of this object as a String.
+  /// Returns a JSON-encoded representation of this object as a `String`.
   func jsonString() throws -> String
```

Comment:
I guess not, should we move the overrides of this into _CustomJSONCodable and use that same pattern?